sdk/python/tests/components/test_python_op.py

```diff
@@ -401,8 +401,15 @@ def test_python_component_decorator(self):
         expected_description = 'Sum component description'
         expected_image = 'org/image'
 
-        @python_component(name=expected_name, description=expected_description, base_image=expected_image)
-        def add_two_numbers_decorated(a: float, b: float) -> float:
+        @python_component(
```

Comment:
I don't see any tests that verify the generated codes of the func that is decorated with python_component?

Reply:
You're right. I'll add the execution of this component.

Reply:
I've made this test verify the execution.